AirCasting/Utils/Constants.swift

```diff
@@ -14,4 +14,9 @@ struct Constants {
         static let dotWidth = 20
         static let dotHeight = 20
     }
+    
+    static let dateFormatter: DateFormatter = {
+        let df = DateFormatter()
+        return df
+    }()
```

Comment:
it could be done like this:
```Swift
static let dateFormatter = DateFormatter()
```

But this is not the way it should work - this `DateFormatter` is a plain formatter shared across the whole app with a name `dateFormatter` ;) it shouldn't be like that, too dangerous. Lets create two formatters - one for mobile-active and one for all other sessions. And put them in `static let`, but not here in `Constants`, but inside `GraphView.swift` so they are private. Smth like this:

```Swift
static let activeMobileSessionDateFormatter: DateFormatter = {
    let df = DateFormatter()
    df.locale = ...
    df.timeZone = ...
    df.dateFormat = ...
    ...
}()
```

Reply:
<img width="705" alt="Screenshot 2021-09-29 at 23 38 43" src="https://user-images.githubusercontent.com/44276070/135352289-cd40d743-3af2-4334-b903-e88e0771d362.png">
I am getting here that kind of error, could you please tell me what to do in this case to make it in the best possible way?